Ten Elephants/Ten Elephants/Models/Meals.swift

```diff
@@ -0,0 +1,116 @@
+//
+//  Meals.swift
+//  Ten Elephants
+//
+//  Created by Дарья Домрачева on 09.12.2021.
+//
+
+import Foundation
+
+struct  Meals: Decodable {
+    enum CodingKeys: String, CodingKey {
+        case meals
+    }
+    let meals: [MealModel]
+}
+
+struct MealModel: Decodable {
```

Comment:
Предложу убрать Model, это dto описатель?
```suggestion
struct Meal: Decodable {
```